packages/perseus-editor/src/widgets/input-number-editor.tsx

```diff
@@ -46,14 +48,12 @@ const answerTypes = {
 
 type Props = {
     value: number;
-    simplify: PropsFor<typeof InputNumber.widget>["simplify"];
-    size: PropsFor<typeof InputNumber.widget>["size"];
-    inexact: PropsFor<typeof InputNumber.widget>["reviewModeRubric"]["inexact"];
```

Comment:
TS was exploding since `reviewModeRubric` could have been null/undefined. This might be an unpopular opinion, but I'm not a big fan of the `PropsFor<typeof SomeReactComponent>` pattern anyway. The editor exists to set widget options, so I feel like this is more correct anyway?

Reply:
I actually agree.  I think the widget options types should be the glue between the editors and their widgets. I think we've gone through iterations of understanding Perseus code and so this might have been me doing this even (probably because the widget option type wasn't exported (yet)).